io-engine/src/bdev/nexus/nexus_io_subsystem.rs

```diff
@@ -238,7 +244,13 @@ impl<'n> NexusIoSubsystem<'n> {
                                 subsystem.get_nqn()
                             );
 
-                            subsystem.resume().await.unwrap();
+                            if let Err(e) = subsystem.resume().await {
```

Comment:
Should we handle these errors?

Reply:
At this moment, it looks like it would require to rework the suspend/resume state machine logic, which seems to be non-trivial.